### DISCUSSION TITLE
How to limit number of pictures taken during a motion event

### DISCUSSION THREAD

**Discussion by mckin21** [Motion-Project/motionplus#63] [Q&A]
I was wondering if there was any way to limit the number of pictures or add a timing interval to the picture taking process during a motion event.

**Comment by Mr-Dave** [comment]
There isn't any option for that.

**Comment by prokrypt** [comment]
A hack I use (for regular Motion, but it's probably the same on MotionPlus) is to leave out the frame counter from the filename so only one picture ends up saved per second.
